drivers/input/Kconfig

```diff
@@ -85,6 +85,33 @@ config INPUT_TOUCHSCREEN
 	bool
 	default n
 
+if INPUT_TOUCHSCREEN && VIDEO_FB
+
+config INPUT_TOUCHSCREEN_SWAPXY
```

Comment:
let's add field to touch_lowerhalf_s, so the multiple touch driver could get the different config.